### ISSUE TITLE
re-enable integration/generate_pipeline_test.go

### ISSUE THREAD

**Issue by aaron-prindle** [GoogleContainerTools/skaffold#7482]
##Description

integration some tests in integration/generate_pipeline_test.go were temporarily disabled due to v2 migration without completing v2 render
Some disabled tests may fail when enabling it due to test skaffold.yaml don't have render config setup in skaffold.yaml, adding that part should fix the issue.
now v2 render is complete, as the goal of v2.0 alpha 1 milestone, we should make all integration tests pass
##ASK

re-enable all tests in integration/generate_pipeline_test.go

**Comment by ericzzzzzzz** [comment]
Hi @aaron-prindle looks like generatePipeline is not implemented in V2. Implementing that seems to require a large effort, should we create another issue for implementing the feature? Or, we don't need this feature in v2?

**Comment by aaron-prindle** [comment]
Yes you are correct, currently `skaffold generate-pipeline` is not implemented for V2.  I think this is lower priority compared to the other tests so now regarding:
Implementing that seems to require a large effort, should we create another issue for implementing the feature? Or, we don't need this feature in v2?

Yes, I think the appropriate thing to do here is to make an issure related to implementing `GeneratePipeline` for V2 and we can focus on it later.  Thanks for investingating this Eric!

I have added the `triage/discuss` label so we can discuss at our triage meeting for relative priority of this and the to be created issue
